marimo/_tutorials/plots.py

```diff
@@ -173,15 +173,12 @@ def __(mo):
         {exponent}
         """
     )
-    return exponent,
+    return (exponent,)
 
 
 @app.cell
 def __(exponent, mo, plt, x):
-    import functools
-
-
-    @functools.cache
```

Comment:
You can check with `_plots.cache_info().hits`, and as was, there would never be a cache hit with functools, since the function would get redefined and clear the cache

I did find that for, mo.cache, private functions did not have a persistent cache- but solved that